requirements.in

```diff
@@ -0,0 +1,6 @@
+buildbot
```

Comment:
I've since learned that this whole file can be shortened to `buildbot[bundle]`.  I'd then like a requirements.txt with pinned versions (`pip install -U -r requirements.in && pip freeze > requirements.txt`, or `pip-compile requirements.in requirements.txt`).

Reply:
Fixed